service/gcs/bridge/bridge.go

```diff
@@ -334,21 +334,37 @@ func (b *bridge) runExternalProcess(message []byte) (*prot.ContainerExecuteProce
 	return response, nil
 }
 
-func (b *bridge) waitOnProcess(message []byte, header *prot.MessageHeader) (*prot.ContainerWaitForProcessResponse, error) {
-	response := &prot.ContainerWaitForProcessResponse{MessageResponseBase: newResponseBase()}
+func (b *bridge) waitOnProcess(message []byte, header *prot.MessageHeader) (response *prot.ContainerWaitForProcessResponse, err error) {
```

Comment:
What is the real issue here? Im wondering if we shouldnt take this case in favor of my async bridge change which removes exit hooks altogether.